test_utils/test_common.h

```diff
@@ -115,6 +115,19 @@
 #pragma warn -8068	/* Constant out of range in comparison. */
 #endif
 
+
+#if defined(__GNUC__) && (__GNUC__ > 2 || \
+			  (__GNUC__ == 2 && __GNUC_MINOR__ >= 7))
+# ifdef __MINGW_PRINTF_FORMAT
+#  define __LA_PRINTF_FORMAT __MINGW_PRINTF_FORMAT
+# else
+#  define __LA_PRINTF_FORMAT __printf__
+# endif
+# define __LA_PRINTFLIKE(f,a)	__attribute__((__format__(__LA_PRINTF_FORMAT, f, a)))
+#else
+# define __LA_PRINTFLIKE(f,a)
+#endif
+
```

Comment:
In addition, please add the following to CMakeLists.txt, line 289:

```cmake
IF("${CMAKE_C_PLATFORM_ID}" MATCHES "MinGW")
  ADD_DEFINITIONS(-D__USE_MINGW_ANSI_STDIO)
ENDIF()
```

It looks like we have it in configure.ac but it is missing in CMakeLists.txt